tests/lib/rules/no-restricted-properties.js

```diff
@@ -75,6 +75,21 @@ ruleTester.run("no-restricted-properties", rule, {
                 object: "obj",
                 property: "foo"
             }]
+        }, {
+            code: "foo.bar",
+            options: [{
+                property: "baz"
+            }]
+        }, {
+            code: "foo.bar",
+            options: [{
+                object: "baz"
+            }]
+        }, {
+            code: "foo()",
```

Comment:
there's no `[[Get]]` there (no property lookup), so i'd expect it to be just as valid as `foo()`